src/games/strategy/triplea/delegate/AbstractEndTurnDelegate.java

```diff
@@ -47,7 +47,6 @@
   public static final String END_TURN_REPORT_STRING = "End of Turn Report for ";
   private boolean m_needToInitialize = true;
   private boolean m_hasPostedTurnSummary = false;
-  private static int CONVOY_BLOCKADE_DICE_SIDES = 6;
```

Comment:
In this case I don't know if it helps. Notice that where it is moved, we have the value "100" 2 lines above. Shouldn't that be a constant as well? But then again, the point of extracting to a constant is one of two reasons:
- assign a label to a value
- consolidate redundant code

When there is no redundant code, IE: the value is used in only one place, then personally I think class level or method level are really about equal. The method level version I think is perhaps a bit simpler for the moment. For the moment we are better off getting rid of the extra class level variables. 

So the casing needs to be updated.